app/containers/Achievements/index.js

```diff
@@ -131,12 +131,13 @@ const Achievements = ({
                     maxCount={achievement.maxCount}
                     factCount={achievement.factCount}
                     currentValue={profile?.highestRating?.rating || null}
-                    lowerValue={achievementsArr[index].lowerValue}
+                    lowerValue={achievement.lowerValue}
```

Comment:
Use lower value that is now coming from subgraph instead of the one hardcoded in the constants.